src/Confluent.SchemaRegistry/SchemaRegistryConfig.cs

```diff
@@ -127,11 +147,71 @@ public string Url
             set { SetObject(SchemaRegistryConfig.PropertyNames.SchemaRegistryRequestTimeoutMs, value.ToString()); }
         }
 
+        /// <summary>
+        ///     File or directory path to CA certificate(s) for verifying the schema registry's key.
+        ///
+        ///     default: ''
+        ///     importance: low
+        /// </summary>
+        public string SslCaLocation
+        {
+            get { return Get(SchemaRegistryConfig.PropertyNames.SslCaLocation); }
+            set { SetObject(SchemaRegistryConfig.PropertyNames.SslCaLocation, value.ToString()); }
+        }
+
+        /// <summary>
+        ///     Path to client's keystore (PKCS#12) used for authentication.
+        ///
+        ///     default: ''
+        ///     importance: low
+        /// </summary>
+        public string SslKeystoreLocation
+        {
+            get { return Get(SchemaRegistryConfig.PropertyNames.SslKeystoreLocation); }
+            set { SetObject(SchemaRegistryConfig.PropertyNames.SslKeystoreLocation, value.ToString()); }
+        }
+
+        /// <summary>
+        ///     Client's keystore (PKCS#12) password.
+        ///
+        ///     default: ''
+        ///     importance: low
+        /// </summary>
+        public string SslKeystorePassword
+        {
+            get { return Get(SchemaRegistryConfig.PropertyNames.SslKeystorePassword); }
+            set { SetObject(SchemaRegistryConfig.PropertyNames.SslKeystorePassword, value.ToString()); }
+        }
+
+        /// <summary>
+        ///     Enable/Disable ssl verify
+        ///
+        ///     default: ''
+        ///     importance: low
+        /// </summary>
+        public bool? SslVerify
+        {
+            get { return GetBool(SchemaRegistryConfig.PropertyNames.SslVerify); }
+            set { SetObject(SchemaRegistryConfig.PropertyNames.SslVerify, value); }
+        }
+
+        /// <summary>
+        ///     Specifies the timeout for requests to Confluent Schema Registry.
+        ///
+        ///     default: 30000
+        /// </summary>
+        [Obsolete("This property will be removed in a future version of this library in favor of the RequestTimeoutMs property")]
+        public int? SchemaRegistryRequestTimeoutMs
```

Comment:
I think this has been accidentally added back in?

Reply:
yes, my mistake
done